urizen/generators/locations/lgd_grid.py

```diff
@@ -0,0 +1,95 @@
+import random
+from urizen.core.map import Map
+from urizen.core.cell_collection import (
+    cell_dungeon_wall,
+    cell_dungeon_floor,
+    cell_dungeon_closed_door,
+    cell_void
+)
+
+
+def lgd_grid_simple(w, h, room_size=4):
```

Comment:
Please add a docstring to this function. More info [here](https://www.python.org/dev/peps/pep-0257/) or [here](https://www.pythonforbeginners.com/basics/python-docstrings/)